### PR TITLE
[TetGen] Make stl reading funtionality availble

### PR DESCRIPTION
**Pull request by j-fu**

### CODE REVIEW

File: T/TetGen/build_tarballs.jl [hunk 1]
```diff
@@ -57,7 +60,7 @@ install_license LICENSE
 """
 
 platforms = supported_platforms(; experimental=true)
-# platforms=[Platform("x86_64", "linux"; libc="glibc")]
+#platforms=[Platform("x86_64", "linux"; libc="glibc")]
```
Comment (giordano) [inline T/TetGen/build_tarballs.jl:63]: What's the point of this line?  Can we just remove it?
```suggestion
```

File: T/TetGen/build_tarballs.jl [hunk 2]
```diff
@@ -57,7 +60,7 @@ install_license LICENSE
 """
 
 platforms = supported_platforms(; experimental=true)
```
[the remark this answers is not in the document]
Comment (j-fu) [reply T/TetGen/build_tarballs.jl]: Ah thanks!

File: T/TetGen/build_tarballs.jl (hunk 1 above)
Comment (j-fu) [reply T/TetGen/build_tarballs.jl:63]: It is for my own testing with TetGen.jl - just as  a note what to do during test cycles in combination with the wrapper package. For this purpose I upload into my own repo for the jll before submitting here.
Can put this elsewhere and remove this.

File: T/TetGen/cwrapper/cwrapper.cxx [hunk 3]
```diff
@@ -182,8 +182,8 @@ extern "C"
     return jl_out;
   }
   
-
-
+  
+
```
[the remark this answers is not in the document]
Comment (j-fu) [reply T/TetGen/cwrapper/cwrapper.cxx:186]: ok...

File: T/TetGen/build_tarballs.jl (hunk 1 above)
Comment (giordano) [reply T/TetGen/build_tarballs.jl]: You can pass the platform you want to build for on the command line: https://docs.binarybuilder.org/stable/#Manually-create-or-edit-build_tarballs.jl  That's what I always do, without touching the script

**Pull request merged**